superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -183,6 +183,8 @@ def _postprocess(self):
             )
             return False
 
+        torch.cuda.empty_cache()
```

Comment:
Is it possible to add a test for this?

Reply:
Thanks, have added tests for it. Please check.